code-challenges.test.js

```diff
@@ -22,11 +33,52 @@ const fibLength2 = 10
 
 // b) Create the function that makes the test pass.
 
+/* Psuedo Code
+   1. Create a new const with the name fibSequence
+   2. pass (number) into fibSequence
+   3.create a new empty array named fib1
+   4. create two new variables num1 and num2, give them each a value of 1
+   5. create an iteration
+   6. create a variable for num3 which is the sum of num1 + num2
+   7. push the iteration into the empty array (fib1)
+   8. at the end of each iteration, change the value of num1 to become num 2 && 
+      num2 to become num3
+   9. return the name of the created array
+*/
+
+
+
+const fibSequence = (number) => {
+  let fib1 = []
```

Comment:
Awesome work on this problem! You can also initialize the fib1 variable to contain both ones.  Ex: `[1, 1]`